test/e2e/persistent_volumes.go

```diff
@@ -555,7 +560,7 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 					Name:    "write-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "touch /mnt/SUCCESS && exit 0 || exit 1"},
+					Args:    []string{"-c", "touch /mnt/SUCCESS && (id -G | grep -P '\b777\b' && exit 0 || exit 1"},
```

Comment:
busybox grep doesn't support -P (PCRE). Use -E (Extended) instead -- \b works with it.